src/tests/resources/TestDynamicPaths.csproj

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I find the name of this project a bit confusing. In what sense are the paths "dynamic". That word is used in several ways in programming languages and NUnit has it's own interpretation as well, so it's a potential source of confusion. 

In fact, the more you know about NUnit, the more confusing it might be! We consider all our test data, for example, to be static because it doesn't change in the course of a test run. We sketched out a bit about what dynamic data would look like (changing live, on the fly) but have not implemented it yet.

OTOH, if the term is essential, we can live with another overload. 😖

Reply:
Rather than 'dynamic' I might call it 'templated.' MSBuild calls it 'expanded.' (https://msdn.microsoft.com/en-us/library/dd997067.aspx)

Reply:
Fixed